city_scrapers/spiders/alle_port_authority.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+import unicodedata
+from datetime import datetime, timedelta
+
+from dateutil.parser import parse
+from lxml import html
+
+from city_scrapers.constants import BOARD, COMMITTEE
+from city_scrapers.spider import Spider
+
+
+class AllePortAuthoritySpider(Spider):
+    name = 'alle_port_authority'
+    agency_name = 'Port Authority of Allegheny County'
+    timezone = 'America/New_York'
+    allowed_domains = ['www.portauthority.org']
+    start_urls = ['https://www.portauthority.org/paac/CompanyInfoProjects/'
+                  'BoardofDirectors/MeetingAgendasResolutions.aspx']
+    event_year = str(datetime.now().year)
+
+    def _build_datatable(self, response):
+        alist_tbody = (response
+                       .xpath('//table[1]/tbody//td')
+                       .extract()
+                       )
+
+        atable = []
+        arow = []
+
+        for item in alist_tbody:
+            tree = html.fragment_fromstring(item)
+            # pdb.set_trace()
+            text = tree.text_content()
+
+            url = tree.xpath('//a/@href')
+            find_att_b = tree.xpath('//b/text()|//strong/text()')
+            if len(find_att_b) >= 1:
+                continue
+            if url:
+                arow.append('{text}: {url}'
```

Comment:
A dictionary with `name` and `url` keys might be more clear here? A dictionary might be a better fit for the rows more generally since you have a sense of what's in each column